### PR TITLE
392 several bugs

### PR DESCRIPTION
**Pull request by lapinlapin**
#### What's this PR do?
Made:
392,382,379,363,362,354,340 
#### How should this be manually tested?
https://localhost:8080/
#### What are the relevant tickets?
#392 
#382 
#379 
#363 
#362 
#354 
#340 
#### URL
Provide a link to your deploy from http://staging-ui.trusty.apasia.tech:8080/branch-392-several-bugs

### CODE REVIEW

**Comment by roma219** [comment]
@lapinlapin check again #392 
Remember to go through all the issues and double-check before submitting a PR.

File: src/components/SortableHeaderItem/SortableHeaderItem.vue [hunk 1]
```diff
@@ -101,6 +101,10 @@ export default {
       type: Boolean,
       required: false,
       default: false
+    },
+    hideHoverArrow: {
```
Comment (roma219) [inline src/components/SortableHeaderItem/SortableHeaderItem.vue]: no

Comment (roma219) [reply src/components/SortableHeaderItem/SortableHeaderItem.vue]: Hide is when sorting is disabled

File: src/views/Markets/MarketsList.vue [hunk 2]
```diff
@@ -21,6 +21,7 @@
           :current-base="currentBase"
           :expand-mode="expandMode"
           :is-favourite="isFavourite(ticker)"
+          :is-first="index === 0"
```
Comment (roma219) [inline src/views/Markets/MarketsList.vue]: no
stop using props for getting basic data such as element position

File: src/views/Markets/MarketsListItem.vue [hunk 3]
```diff
@@ -7,6 +7,7 @@
     :is-favourite="isFavourite"
     :market-cap="marketCap"
     :expand-mode="expandMode"
+    :is-first="isFirst"
```
Comment (roma219) [inline src/views/Markets/MarketsListItem.vue]: no

File: src/views/Markets/MarketsListItem.vue [hunk 4]
```diff
@@ -35,6 +36,10 @@ export default {
     isFavourite: {
       type: Boolean,
       default: false
+    },
+    isFirst: {
```
Comment (roma219) [inline src/views/Markets/MarketsListItem.vue]: no

File: src/views/Markets/MarketsListItemPrices.vue [hunk 5]
```diff
@@ -2,6 +2,9 @@
   <div :class="{'ticker-list-row_expanded': expandMode}">
     <div
       v-if="!expandMode"
+      :class="{
+        'tickers-list-row--first': isFirst
```
Comment (roma219) [inline src/views/Markets/MarketsListItemPrices.vue]: 🤦‍♂️ 
Simply use CSS, `:first-child`, no need to create this complex props passing

File: src/views/OrderBook/OrderBookTable.vue [hunk 6]
```diff
@@ -15,6 +15,7 @@
       :header-right-padding="tableType === 'sell' ? 1 : 0.6"
       :default-sort="defaultSort"
       :empty-area="true"
+      :hide-hover-arrow="true"
```
Comment (roma219) [inline src/views/OrderBook/OrderBookTable.vue]: no

File: src/views/UnlockAccountPopup/UnlockAccountPopup.vue [hunk 7]
```diff
@@ -77,9 +76,14 @@ export default {
...
   methods: {
-    ...mapActions('acc', ['unlockWallet', 'cloudLogin']),
+    ...mapActions({
+      unlockWallet: 'acc/unlockWallet',
+      cloudLogin: 'acc/cloudLogin',
+      hideWithdrawModal: 'withdraw/toggleModal'
+    }),
 
     showModal() {
+      this.hideWithdrawModal()
```
Comment (roma219) [inline src/views/UnlockAccountPopup/UnlockAccountPopup.vue]: wtf is this??? This component is for unlocking account, it shouldn't do anything about withdraw/deposit/lalala anything else.

File: src/views/Mobile/MobileOrders.vue [hunk 8]
```diff
@@ -114,6 +114,16 @@ export default {
     top: 0.4875rem;
     z-index: 100;
   }
+  .mobile-orders-chart {
+    width: 100%;
+    height: 230px;
+    margin-top: 40px;
+    z-index: 100;
+    position: absolute;
```
Comment (roma219) [inline src/views/Mobile/MobileOrders.vue]: No position-absolute
No pixels

File: src/views/Mobile/MobileOrders.vue [hunk 9]
```diff
@@ -114,6 +114,16 @@ export default {
...
...vQK7V2HbZ4aeog6Zqm7WddZehyVOWtfcp8i/IqsvfLsPObRqIOetStg0nXlbf+Y7X9+hL3IAiCoN2EuAdBEMwgi8RdsE6evcjf2jyfIru22fiiZW0rS6Ntnp9n85br4OXnYX28sCzbsPiKrSrcNJ1OMwg6f43no21l2Hy89Nanjs3GSbz203FCWVwRkqZJOp3GS2vDRTbBy6cobO2WsjhNlZ9ej16u8tF+NlxFkW9ZPhKn43XYiy+iyk/HFflaO8uuuAdBEATTTYh7EATBDBLiHgRBMIOEuAdBEMwgIe5BEAQzyIK4z8/5DkEQBMH0EeIeBEEwg0xM3Gf9ZBIny+km2m8xUR/ltLF+RiruZXnauKbr9/yxNc1nVEyyHFXrHmY9taW+h02bt2vUZfPyH9U6x1XPs1j+qnUvEvcmkNizQ1lcP0h+RfliL4urYy/Loy5e+jp5Nll3v2WsWw7PXkWT8tdhmHl5NM1/2P7EW586aTx7FWXpvHI0oSit2PvJ25apKo8mvlDHR4N/URq77jp5V/mUxddZh/ZZEPeucQqCIAimlxD3IAiCGSTEPQiCYAapJe7ECXVtXpwse1h/i8TXoSy9Dut4zy5pPF8vzvMTu/Yt8gPtW+XnLUvYSyt2Ha/9bLxGfIqwvjasbTosyzq+CJvGxnl2idPL2q9ouYgqH4kv8rHx+r+2N0Gnteh46y/hKsRf0tj0RXaPsnQ2rbbZOMFLV0aVf1m8jvP8ymxN47SPXi7y92wL4t4xEUEQBMH0EuIeBEEwg4S4B0EQzCCtEXfWHyeYoIzYP4JpY5L77CJx1wWRsEYnFJ8qW1n6oviicJHNxpVh/XU6Lw8bL2gfHWftgk4rfnrZ+nk2ba8TlmWN9vN8itJZdD4S9pbrIP42j7Kw2HRY28vS2rC2e2HP7qUXitIJkt7zs8vWR2wSroP199J6eXo2sVtbXYrWIWib9rPoeLts0+tljfYVH73s+Wm7jquy67D1t3FefFnYs7O8RNzFwVv28OKtjXA/NhtfZPOweYmtKCzLnq0fSDtIehg0vcbLqyh/bbfLRXE2bOOq7EKdeEHC1mcQbP5lDMOvbh5QlYeOt+GmVKWV/PtZR910/ebvIXmV5VlnXVU+Nt6uz4YHwctL21xxD4IgCKabXNz5EwRBEMwSh7O/BUM2KhhXaEgJAAAAAElFTkSuQmCC);
```
Comment (roma219) [inline src/views/Mobile/MobileOrders.vue]: wtf is this? Just use an image normally, there is a folder for images in project

**Review by roma219 — COMMENTED** [review]
Needs changes

Info for study:
Read http://htmlbook.ru/css/last-child, http://htmlbook.ru/css/first-child 
And stop using props for this puprpose

File: src/components/SortableTable/SortableTable.vue [hunk 10]
```diff
@@ -14,6 +14,7 @@
         :padding-left="header.paddingLeft + 0.56"
         :large="large"
         :disable-sort="header.disableSort"
+        :hide-hover-arrow="hideHoverArrow"
```
Comment (roma219) [inline src/components/SortableTable/SortableTable.vue]: Check every string in "Changes" before submitting a PR for unused things like that

File: src/views/NewOrder/NewOrder.vue [hunk 11]
```diff
@@ -208,6 +222,14 @@ export default {
       showConfirm: 'newOrder/showConfirm',
       hideConfirm: 'newOrder/hideConfirm'
     }),
+    incPrice() {
```
Comment (roma219) [inline src/views/NewOrder/NewOrder.vue]: Stop doing things from different issues here, please remove

File: src/views/NewOrder/NewOrder.vue [hunk 12]
```diff
@@ -263,6 +285,44 @@ export default {
     display: flex;
     flex-direction: column;
 
+    .new-order-decrement {
```
Comment (roma219) [inline src/views/NewOrder/NewOrder.vue]: remove

File: src/views/NewOrder/NewOrder.vue [hunk 13]
```diff
@@ -283,4 +343,14 @@ export default {
       justify-content: center;
     }
   }
+  @media screen and (max-width: 800px) {
```
Comment (roma219) [inline src/views/NewOrder/NewOrder.vue]: remove

**Review by roma219 — CHANGES_REQUESTED** [review]
Again - double check every issues you are doing BEFORE submiting a PR - see #392 

STOP doing things not listed in specified issues - remove price plus/minus

Cleanup

Also:
Horizontall scroll
image

**Comment by roma219** [comment]
@lapinlapin What is not understandable about that:
> @lapinlapin check again #392
Remember to go through all the issues and double-check before submitting a PR.

**Comment by lapinlapin** [comment]
Check it.
1. Revert Confirm Order modal to be wider & display values in column on mobile - **FIXED**
2. New Order -> Select Market tab -> select Sell - **FIXED**
3. Errors - **FIXED**
4. this screen does not exist already
5. Double border - **FIXED**
6. Confirm order modal reversed assets - **FIXED**
7. Get and Spend Inputs are too far away, make it a bit closer - **FIXED**

File: src/components/SortableTable/SortableTable.vue [hunk 14]
```diff
@@ -79,6 +79,10 @@ export default {
     columnsConfig: {
       type: Array,
       default: () => []
+    },
+    hideHoverArrow: {
```
Comment (roma219) [inline src/components/SortableTable/SortableTable.vue]: Clean up means: check ALL of your changes, not only 1 specific file/place

**Review by roma219 — CHANGES_REQUESTED** [review]
Cleanup not complete

Not fixed (no overlay):
image

@lapinlapin Yes it does exist, still no scroll:
image

**Comment by roma219** [comment]
@lapinlapin

File: src/views/ActiveOrders/ActiveOrdersTable.vue [hunk 15]
```diff
@@ -67,4 +67,9 @@ export default {
   .active-orders-table {
     height: 100%;
   }
+  @media (max-width: 50rem) {
```
Comment (roma219) [inline src/views/ActiveOrders/ActiveOrdersTable.vue:70]: wtf is this? @lapinlapin

**Review by roma219 — APPROVED** [review]
@lapinlapin Again: you should check your code & all the tickets yourself FIRST, not just throw it to review hoping it gonna pass.

File: src/views/ActiveOrders/ActiveOrdersTable.vue (hunk 15 above)
Comment (lapinlapin) [reply src/views/ActiveOrders/ActiveOrdersTable.vue:70]: in the mobile view appears the menu, so the height of the container should be reduced by the height of the menu

Comment (roma219) [reply src/views/ActiveOrders/ActiveOrdersTable.vue:70]: image

**Pull request merged**